src/native/api.c

```diff
@@ -126,7 +126,7 @@ void llhttp_reset(llhttp_t* parser) {
   llhttp_type_t type = parser->type;
   const llhttp_settings_t* settings = parser->settings;
   void* data = parser->data;
-  uint8_t lenient_flags = parser->lenient_flags;
+  uint16_t lenient_flags = parser->lenient_flags;
```

Comment:
Maybe it should be `__typeof(parser->lenient_flags)` since it could grow bigger in the future?

Reply:
Good point, but for now I wouldn't worry about it too much. It will take a lot for other 8 lenient flags to appear, I hope.